SystemAudioPlayer/SystemAudioPlayerImplementation.cpp

```diff
@@ -104,14 +104,24 @@ namespace Plugin {
         SourceType sourceType;
         PlayMode playMode;
         std::string s_audioType,s_sourceType,s_playMode;
-      
         s_audioType = parameters["audiotype"].String(); 
```

Comment:
can we try to avoid the extra string variables s_audioType, s_sourceType & s_playMode?

auto audioType = audioTypeFromString(parameters["audiotype"].String())
if (audioType == AudioType::AudioType_None) {
            SAPLOG_INFO("SystemAudioPlayerImplementation Open Audiotype :%s is not supported",s_audioType.c_str());
            returnResponse(false);
}

auto sourceType = sourceTypeFromString(parameters["sourcetype"].String());
...

Reply:
done